<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPollDialog</class>
 <widget class="QDialog" name="NewPollDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Poll - Mattermost</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="questionLabel">
     <property name="text">
      <string>Question</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="questionValue"/>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="option1Label">
     <property name="text">
      <string>Option 1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="option1Value"/>
   </item>
   <item>
    <widget class="QLabel" name="option2Label">
     <property name="text">
      <string>Option 2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="option2Value"/>
   </item>
   <item>
    <widget class="QLabel" name="option3Label">
     <property name="text">
      <string>Option 3 (not required)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="option3Value"/>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxAnonymous">
     <property name="text">
      <string>Anonymous</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxProgress">
     <property name="text">
      <string>Progress</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxAllowAdditional">
     <property name="text">
      <string>Allow all users to add additional options</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewPollDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewPollDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
